pkg/services/contexthandler/contexthandler.go

```diff
@@ -110,6 +111,35 @@ func FromContext(c context.Context) *contextmodel.ReqContext {
 	return nil
 }
 
+// CopyWithReqContext returns a copy of the parent context with a cloned version of its RequestContext value
```

Comment:
Incorrect name of function

Reply:
Fixed func name here [9aea1b1](https://github.com/grafana/grafana/pull/70100/commits/9aea1b1812c197a8e6a406a14bb64e1559d2daab)